Kinvey/KinveyTests/UserTests.swift

```diff
@@ -5122,115 +5122,115 @@ extension UserTests {
-                    case .requestTimeout:
-                        XCTAssertEqual(error.description, "Request Timeout")
-                    default:
-                        XCTFail(error.localizedDescription)
-                    }
-                }
-            }
-
-            expectationLogin?.fulfill()
-        }
-
-        waitForExpectations(timeout: defaultTimeout) { error in
-            expectationLogin = nil
-        }
-    }
-    
-    func testMICCancelUserAction() {
-        var runLoop: CFRunLoop?
-        defer {
-            if let runLoop = runLoop {
-                CFRunLoopStop(runLoop)
-            }
-        }
-        mockResponse { (request) -> HttpResponse in
-            runLoop = CFRunLoopGetCurrent()
-            RunLoop.current.run(until: Date(timeIntervalSinceNow: 10))
-            DispatchQueue.main.async {
-                runLoop = nil
-                guard let keyWindow = UIApplication.shared.keyWindow,
-                    let navigationController = keyWindow.rootViewController as? UINavigationController,
-                    let navigationController2 = navigationController.topViewController?.presentedViewController as? UINavigationController,
-                    let micLoginViewController = navigationController2.topViewController as? Kinvey.MICLoginViewController,
-                    let closeButton = micLoginViewController.navigationItem.leftBarButtonItem
-                else {
-                    XCTFail("Kinvey.MICLoginViewController close button not found")
-                    return
-                }
-                closeButton.target!.performSelector(onMainThread: closeButton.action!, with: self, waitUntilDone: true)
-            }
-            return HttpResponse(error: timeoutError)
-        }
-        defer {
-            setURLProtocol(nil)
-        }
-        
-        weak var expectationLogin = expectation(description: "Login")
-        
-        let redirectURI = URL(string: "throwAnError://")!
-        User.presentMICViewController(
-            redirectURI: redirectURI,
-            micUserInterface: .uiWebView,
-            options: try! Options(timeout: 60)
-        ) {
-            XCTAssertTrue(Thread.isMainThread)
-            switch $0 {
-            case .success:
-                XCTFail("A failure result is expected")
-            case .failure(let error):
-                XCTAssertTrue(error is Kinvey.Error)
-                if let error = error as? Kinvey.Error {
-                    switch error {
-                    case .requestCancelled:
-                        break
-                    default:
-                        XCTFail(error.localizedDescription)
-                    }
-                }
-            }
-            
-            expectationLogin?.fulfill()
-        }
-        
-        waitForExpectations(timeout: defaultTimeout) { error in
-            expectationLogin = nil
-        }
-    }
+//    func testMICTimeoutAction() {
```

Comment:
Is there a good reason for keeping this commented test, or should we delete it completely?

Reply:
decided to remove UIWebView completely and all tests related, which will cause a breaking change, so a major release will be necessary